docs/API.md

```diff
@@ -287,12 +287,10 @@ A default installation will accept the following `language tags`:
 * `sv` or `sv_SE` for Swedish language.
 
 
-### Unsigned integer
+### Non-negative integer
 
 Basic data type: number (integer)
  
-An unsigned integer is either positive or zero.
```

Comment:
Why removing this line ? It could be nice to keep it, so it's clear what we mean by `non-negative integer`.

Reply:
I was changing all "unsigned" to "non-negative" and all "positive" to "strictly positive". With those updates the statement became "A non-negative integer is either zero or strictly positive.", which felt to me like a non-statement. So I removed it. Do you think that that would be helpful to anyone? If you do I'll add it back in.

Reply:
I added the updated clarification.

Reply:
I think it's always better to clarify our choices, thanks for the update.